streaming/index.js

```diff
@@ -525,9 +526,11 @@ const startWorker = (workerId) => {
       ws.isAlive = true;
     });
 
+    let channel;
```

Comment:
Not sure I understand this change. Why make it mutable? We never fall through any of the switch cases, right?

Reply:
A few of the switch cases needed to reuse the same assembled string (channel) in two places at once, but a switch clause is not a separate scope, so you can't redefine a const there. Therefore, it's easiest to define the channel up there.